src/psyclone/core/variables_access_info.py

```diff
@@ -161,11 +171,11 @@ def options(self, key=None):
 
         '''
         if key:
-            if key not in VariablesAccessInfo._VALID_OPTIONS:
+            if key not in VariablesAccessInfo._DEFAULT_OPTIONS:
+                valids = list(VariablesAccessInfo._DEFAULT_OPTIONS.keys())
+                valids.sort()
```

Comment:
Probably worth a comment explaining that `sort` is needed for reproducible test output.

Reply:
Done.